cypress/integration/teacher-training-adviser.spec.js

```diff
@@ -570,15 +574,20 @@ describe("Get-into-teaching - teachet training adviser flow", () => {
 		cy.whenDoYouWantToStartYourTeacherTraining("Equivalent-2021");
 		cy.typeDateOfBirth("22", "08", "2000", true);
 		cy.whereDoYouLive("UK", false, true);
-		cy.get("#equivalent-uk-candidate-address-line1-field").type("25");
-		cy.get("#equivalent-uk-candidate-address-line2-field").type(
+		cy.get("#teacher-training-adviser-steps-uk-address-address-line1-field").type("25");
+		cy.get("#teacher-training-adviser-steps-uk-address-address-line2-field").type(
 			"Delbury Court"
 		);
-		cy.get("#equivalent-uk-candidate-address-city-field").type("Telford");
-		cy.get("#equivalent-uk-candidate-address-postcode-field").type("TF3 2BP");
+		cy.get("#teacher-training-adviser-steps-uk-address-address-city-field").type("Telford");
+		cy.get("#teacher-training-adviser-steps-uk-address-address-postcode-field").type("TF3 2BP");
 		cy.clickOnContinueButton();
 		cy.enterUKTelephoneNumber("0123454748", true);
 
+		cy.get(".govuk-fieldset__heading")
+			.should("exist")
+			.should("have.text", "You told us you live in the United Kingdom");
```

Comment:
This is a test change - as far as I can tell we should be showing the callback step for equivalent candidates (uk in this case)